duo-client/src/test/java/com/duosecurity/client/HttpRateLimitRetryTest.java

```diff
@@ -105,8 +105,8 @@ public Call answer(InvocationOnMock invocationOnMock) throws Throwable {
         });
 
         Response actualRes = http.executeHttpRequest();
-        assertEquals(Http.MAX_REQUEST_ATTEMPTS, responses.size());
-        assertEquals(Http.MAX_REQUEST_ATTEMPTS, calls.size());
+        assertEquals(7, responses.size());
```

Comment:
Can you do "future us" a favor and add a comment here (and one on line 119) as to why we expect 7 and 6?